reg_tests/global_cycle/C768.fv3gfs.sh

```diff
@@ -33,6 +33,7 @@ export LONB=3072
 export LATB=1536
 
 export FIXgsm=$BASE_GSM/fix/fix_am
+export FNAISC=$FIXgsm/CFSR.SEAICE.1982.2012.monthly.clim.grb
```

Comment:
Should this be the `IMS-NIC.blended.ice.monthly.clim.grb` or are you specifically using the prior one here?

Reply:
Ah, gotcha. Yes the GFSv16 is using the older (`CFSR.SEAICE.1982.2012.monthly.clim.grb`) file.